lms/lib/courseware_search/lms_filter_generator.py

```diff
@@ -0,0 +1,21 @@
+"""
+This file contains implementation override of SearchFilterGenerator which will allow
+    * Filter by all courses in which the user is enrolled in
+"""
+
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
```

Comment:
This seems to be unused - if it were it would likely be the wrong class to use anyway, because it is specific to / separated ids, which are not used in split mongo